marine.c

```diff
@@ -966,6 +977,374 @@ WS_DLL_PUBLIC void marine_free(marine_result *ptr) {
     }
 }
 
+static void marine_packet_field_join_children_with_duplicate_names(marine_packet_field *field);
```

Comment:
Don't declare functions in the middle of the file, declare all of them in the same place (Next to where `reset_epan_mem` is declared, for example). Applies to a few more functions below I think.

Reply:
Fixed.